packages/components/src/components/hds/app-header/index.hbs

```diff
@@ -0,0 +1,34 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: MPL-2.0
+}}
+
+<div class={{this.classNames}} ...attributes>
+  {{!-- GlobalItems --}}
+  <div class="hds-app-header__sub-section">
+    {{! Intended for: ProductMenu, HomeLink/Logo }}
+    {{yield to="globalItemsBefore"}}
+    {{! Intended for: OrgPicker }}
+    <div class="hds-app-header__global-items-mobile-view">
+      {{yield to="globalItemsAfter"}}
+    </div>
+  </div>
+
+  {{!-- UtilityItems --}}
+  <div class="hds-app-header__sub-section">
+    {{!-- Button appears in mobile/tablet view (hidden in desktop view) --}}
+    <Hds::AppHeader::MenuButton 
+      @onClickToggle={{this.onClickToggle}} 
+      @isOpen={{this.isOpen}} 
+      @menuContentId={{this.menuContentId}} 
+    />
+
+    {{! Intended for: Search, HelpMenu, GeoPicker, UserMenu }}
+    <div class="hds-app-header__menu" id={{this.menuContentId}}>
+      <div class="hds-app-header__global-items-desktop-view">
```

Comment:
Not sure if these are the best class names:
"hds-app-header__global-items-mobile-view"
& "hds-app-header__global-items-mobile-view" (used above within globalItems)

Reply:
Is this essentially handling the "conditional" rendering of the elements based on the viewport?

Reply:
Updated now but don't know if the new names are much better...